fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
OSGi trumps @VisibleForTesting.
...but please leave a comment or the like in the code to communicate the same idea.